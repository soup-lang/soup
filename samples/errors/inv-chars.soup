if `false
    write(true?)

write(false);
