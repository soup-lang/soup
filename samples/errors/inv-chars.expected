samples/errors/inv-chars.soup:1
    illegal token: `
samples/errors/inv-chars.soup:2
    illegal token: ?
samples/errors/inv-chars.soup:4
    illegal token: ;
